content/themes/wcmc-beijing/blocks/block-contact/block.php

```diff
@@ -6,4 +6,109 @@
+$contactNumberValue = get_field( 'contact_number', 'option' );
+$contactEmailTitle = __( 'Email Address', 'wcmc' );
+$contactEmailDescription = get_field( 'email_address_instructions', 'option' );
+$contactEmailAddress = get_field( 'email_address', 'option' );
+
+$formTitle = __( 'Contact Form', 'wcmc' );
+?>
+
+<div class="contact">
+  <div class="contact__columns">
+    <div class="contact__column">
+      <?php if ($contactDetailsTitle): ?>
+        <h3 class="contact__title">
+          <?php echo $contactDetailsTitle; ?>
+        </h3>
+      <?php endif; ?>
+      <ul class="contact__items">
+        <li
+          v-if="contactDetails.address"
+          class="contact__item"
+        >
+          <?php if ($contactAddressTitle): ?>
+            <h4 class="contact__subtitle">
+              <?php echo $contactAddressTitle; ?>
+            </h4>
+          <?php endif; ?>
+          <?php if ($contactAddress): ?>
+            <p class="contact__text" >
+              <?php echo $contactAddress; ?>
+            </p>
+          <?php endif; ?>
+        </li>
+        <?php if ($contactNumberDescription || $contactNumberValue): ?>
+          <li class="contact__item">
+            <?php if ($contactNumberTitle): ?>
+              <h4 class="contact__subtitle">
+                <?php echo $contactNumberTitle; ?>
+              </h4>
+            <?php endif; ?>
+            <?php if ($contactNumberDescription): ?>
+              <p class="contact__text">
+                <?php echo $contactNumberDescription; ?>
+              </p>
+            <?php endif; ?>
+            <?php if ($contactNumberValue): ?>
+              <a class="contact__link" href="<?php echo $contactNumberValue; ?>">
+                <?php echo $contactNumberValue; ?>
+              </a>
+            <?php endif; ?>
+          </li>
+        <?php endif; ?>
+        <?php if ($contactEmailDescription || $contactEmailAddress): ?>
+          <li class="contact__item">
+            <?php if ($contactEmailTitle): ?>
+              <h4 class="contact__subtitle">
+                <?php echo $contactEmailTitle; ?>
+              </h4>
+            <?php endif; ?>
+            <?php if ($contactEmailDescription): ?>
+              <p class="contact__text">
+                <?php echo $contactEmailDescription; ?>
+              </p>
+            <?php endif; ?>
+            <?php if ($contactEmailAddress): ?>
+              <a class="contact__link" :href="mailto:<?php echo $contactEmailAddress; ?>">
+                <?php echo $contactEmailAddress; ?>
+              </a>
+            <?php endif; ?>
+          </li>
+        <?php endif; ?>
+      </ul>
+    </div>
+    <div class="contact__column">
+      <div class="form-contact">
+        <div class="form-contact__content">
+          <h3 class="form-contact__title">
+            <?php echo $formTitle; ?>
+          </h3>
+          <form class="app-form" disabled>
+            <label for="your-name" class="app-form__label">
```

Comment:
Can we look at this together to see if we can make this dynamic?

Reply:
Can we make it dynamic in a new PR later on?